content/docs/getting-started.md

```diff
@@ -18,109 +18,109 @@ redirect_from:
   - "docs/environments.html"
 ---
 
-This page is an overview of the React documentation and related resources.
+Ta podstrona to przegląd dokumentacji Reacta oraz powiązanych zasobów.
 
-**React** is a JavaScript library for building user interfaces. Learn what React is all about on [our homepage](/) or [in the tutorial](/tutorial/tutorial.html).
+**React** to biblioteka JavaScript służąca do budowy interfejsów. Dowiedz się, czym tak naprawdę jest React, na naszej [stronie głównej](/) lub [w tutorialu](/tutorial/tutorial.html).
 
 ---
 
-- [Try React](#try-react)
-- [Learn React](#learn-react)
-- [Staying Informed](#staying-informed)
-- [Versioned Documentation](#versioned-documentation)
-- [Something Missing?](#something-missing)
+- [Wypróbuj Reacta](#try-react)
+- [Ucz się Reacta](#learn-react)
+- [Bądź na bieżąco](#staying-informed)
+- [Dokumentacje wersji](#versioned-documentation)
+- [Czegoś brakuje?](#something-missing)
 
-## Try React {#try-react}
+## Wypróbuj Reacta {#try-react}
 
-React has been designed from the start for gradual adoption, and **you can use as little or as much React as you need.** Whether you want to get a taste of React, add some interactivity to a simple HTML page, or start a complex React-powered app, the links in this section will help you get started.
+React został zaprojektowany z myślą o stopniowym wdrażaniu. Dzięki temu możesz użyć **tak wiele lub tak mało Reacta, jak tego potrzebujesz.** W zależności czy potrzebujesz jedynie spróbować Reacta i dodać trochę interaktywności do prostej strony HTML, czy zacząć skomplikowaną aplikację zasilaną Reactem, linki w tej sekcji pomogą Ci zacząć.
 
 ### Online Playgrounds {#online-playgrounds}
 
-If you're interested in playing around with React, you can use an online code playground. Try a Hello World template on [CodePen](codepen://hello-world) or [CodeSandbox](https://codesandbox.io/s/new).
+Jeśli jesteś zainteresowany wypróbowaniem Reacta, możesz w tym celu użyć playgroundów online. Wypróbuj szablon Hello World na [CodePen](codepen://hello-world) lub [CodeSandbox](https://codesandbox.io/s/new).
```

Comment:
`playgroundów` - zostawiamy to słowo czy tłumaczymy na `piaskownic` ?

Reply:
@jakubdrozdek rozwiń proszę myśl nie zgadzasz się czyli jesteś za pozostawieniem `playgroundów` ❓

Reply:
a czy czasem Piaskownica === Sandbox bardziej niż Piaskownica === Playground?

Reply:
W tym kontekście playground jest tożsame z sandbox. W kwadratowych nawiasach masz nawet odnośnik do `[CodeSandbox]`. Dla `playground` nie mamy polskiego odnośnika(przynajmniej ja takiego nie znam), który byłby powszechnie znany w literaturze IT. @jakubdrozdek @TeoTN @cvqprs  co o tym sądzisz?

Reply:
@bartlomiejzuber W pierwszym lepszym słowniku znalazłem, że "sandbox" jest tłumaczone jako ["środowisko testowe"](https://pl.bab.la/slownik/angielski-polski/sandbox).
Jednak dla mnie jako developera to określenie bardziej kojarzy się z etapem rozwoju oprogramowania... ;-)
Spotkałem się też z "trybem piaskownicy" np. przy okazji oprogramowania antywirusowego (ogólnie: środowiska odizolowanego, bez wpływu na inne części systemu) ([przykłady użycia na linguee.pl](https://www.linguee.pl/angielski-polski/t%C5%82umaczenie/sandbox+mode.html)).
Może nasz nowy kolega Marcin pomoże? :-)

Reply:
Ok, zostawiam póki co, aż ktoś nie zaproponuje czegoś lepszego, tak?

Reply:
a może "środowisko doświadczalne"?

Reply:
"Środowisko doświadczalne" też mi pasuje.